lib/plugins/aws/deploy/compile/events/apiGateway/lib/methods.js

```diff
@@ -351,7 +383,7 @@ module.exports = {
                 "RequestParameters" : {},
                 "Integration" : {
                   "IntegrationHttpMethod" : "POST",
-                  "Type" : "AWS",
+                  "Type" : "${integrationType}",
```

Comment:
The entire template is wrapped in back-ticks as if it's a complete JSON file, so this would work 😊